tests/db_tests/resources/db_seed_data.py

```diff
@@ -17,4 +17,13 @@
     "organisation_id": [str(uid["uuid1"]), str(uid["uuid1"]), str(uid["uuid3"])],
 }
 
-# TODO add further test data for all models. Need to generate some new UUIDs for use in further table fk lookups.
+PACKAGE_DATA = {
+    "id": [uid["uuid7"], uid["uuid8"], uid["uuid9"]],
+    "package_name": ["Regeneration Project", "North Eastern Scheme", "Southern Access Project"],
+    "fund_type_id": ["HIJ", "LMN", "OPQR"],
+    "organisation_id": [str(uid["uuid1"]), str(uid["uuid2"]), str(uid["uuid3"])],
+    "name_contact_id": [str(uid["uuid4"]), str(uid["uuid5"]), str(uid["uuid6"])],
+    "project_sro_contact_id": [str(uid["uuid10"]), str(uid["uuid11"]), str(uid["uuid12"])],
```

Comment:
The last 3 fields in this table (project_sro_contact_id, cfo_contact_id, m_and_e_contact_id) the fk's need to reference a valid PK of Contact (see model definition of these fields for fk relations).

So according to the pre-defined CONTACT_DATA they will need to be in the range "uuid4" to "uuid6". These Contact rows can be referenced as many times as we want, so you can re-use the keys multiple times (since it's a many-to-one).

Ordinarily the data here would result in a FK ref not found type error, but it's turned off on SQLite as default (I'll fix this shortly)